subworkflows/local/gene_annotation.nf

```diff
@@ -0,0 +1,67 @@
+// INCLUDING MODULES
+// Importing the necessary modules for the workflow
+include { BAKTA_BAKTA } from '../../modules/nf-core/bakta/bakta/main' 
+include { BAKTA_BAKTADBDOWNLOAD } from '../../modules/nf-core/bakta/baktadbdownload/main'
+include { PROKKA } from '../../modules/nf-core/prokka/main'                                                                          
+
+// Defining the workflow
+workflow GENE_ANNOTATION{
+    // Defining the input channel
+    take: 
+        ch_genome
+
+    // Defining the main process
+    main:
+        // Initializing empty channels for the output files
+        prokka_gff = Channel.empty()
+        bakta_gff = Channel.empty()
+        ch_versions = Channel.empty()
+
+        // Checking if the prodigal training file is provided
+        if (!params.prodigal_training_file){
+            prodigal = []
+        }
+        else{
+            prodigals = file(params.prodigal_training_file, checkIfExists: true)
```

Comment:
`prodigal_training_file` parameter should be initialized in `nextflow.config`

Reply:
resolved by [c6d85a7](https://github.com/cidgoh/nf-seqqc/pull/44/commits/c6d85a7a7fac79b49086c954d56d6b8dd1251216)